views/layouts/login-layout.pug

```diff
@@ -27,10 +27,6 @@ html
           .navbar-right
             a.btn.btn-primary.navbar-btn(href='/accounts/register', role='button') Sign Up
 
-
-    .container
```

Comment:
Oh nvm I see - the layout naming is a little misleading now though.  Any thoughts on a way to change it to clarify purpose of layout?